scripts/build.py

```diff
@@ -81,13 +81,13 @@ def main():
     for lang in optional_languages:
         group = parser.add_mutually_exclusive_group()
         group.add_argument(
-            '--disable-{}'.format(lang),
+            '--disable_{}'.format(lang),
```

Comment:
Why the change to `_`?

Reply:
eventual compatibility with absl and avoiding confusion with script naming. Happy to update if you do not like it

Reply:
No strong feelings. Just curious.